helpers/argumentParsers.go

```diff
@@ -1766,3 +1766,133 @@ func ParseSocketOption(rawValue uint64) (SocketOptionArgument, error) {
 	}
 	return v, nil
 }
+
+type BpfProgTypeArgument int32
```

Comment:
I believe we have a way to get program type added in libbpfgo already:

https://github.com/aquasecurity/libbpfgo/blob/216bd5b702c8c8867d2a431f7050740b24c3a152/libbpfgo.go#L1127-L1128

wouldn't that interface be enough ?